core/wiki/macros/list.tid

```diff
@@ -72,11 +76,12 @@ tags: $:/tags/Macro
 </$set>
 \end
 
-\define list-tagged-draggable(tag,subFilter,emptyMessage,itemTemplate,elementTag:"div")
+\define list-tagged-draggable(tag,subFilter,type:"ul",subtype:"li",class:"",emptyMessage,itemTemplate,elementTag:"div")
```

Comment:
this would make it the same as list-links-dragable. ... So we have 2 macros that do the same thing. IMO that's wrong!